components/TrackerDash/ResultTablePane.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'underscore';
 import $ from 'jquery';
-import * as d3 from 'd3';
+import { median } from 'd3-array';
```

Comment:
I feel a bit uneasy about mixing d3v3 and d3v4 in TrackerDash.  Is there any reason why we shouldn't consistently use d3v3 for this component?